README.md

```diff
@@ -73,7 +73,8 @@ jobs:
 | asset_name | **Optional** | Customize asset name if do not want to use the default format `${BINARY_NAME}-${RELEASE_TAG}-${GOOS}-${GOARCH}`. <br>Make sure set it correctly, especially for matrix usage that you have to append `-${{ matrix.goos }}-${{ matrix.goarch }}`. A valid example could be  `asset_name: binary-name-${{ matrix.goos }}-${{ matrix.goarch }}`. |
 | retry | **Optional** | How many times retrying if upload fails. `3` by default. |
 | post_command | **Optional** | Extra command that will be executed for teardown work. e.g. you can use it to upload artifacts to AWS s3 or aliyun OSS |
-| compress_assets | **Optional** | Upload execuable binaries rather than `.tar.gz` or `.zip` package. |
+| compress_assets | **Optional** | Upload executable binaries rather than `.tar.gz` or `.zip` package. |
+| compress_assets_format | **Optional** |  Values of `zip` or `gz` will produce executable binaries in the specified format . `""` default. By default linux binaries will be `gz` format and windows binaries will be `zip`|
```

Comment:
Sorry but change exist var is not accpetable since we need to keep forward compaitibility. Would you please still use name `compress_assets` and make sure `TRUE` has same behavior as before? The only thing need to add is another value `zip` to forcely use `zip` on all platforms.

Reply:
When I started working to try to extend `compress_assets` it occurred to me that it might break compatibility for people who are already using the plugin.

Do you want `compress_assets` to take 3 values? `TRUE`, `FALSE, and `zip`?  I have a concern that it seems like bad practice to have the boolean `compress_assets` take both boolean and a string value (ie `zip`)

Reply:
Yes. I think we can extend `compress_assets` to take several values:       
- `auto`(default): windows uses `zip` and others uses `tar.gz`, which has same behavior of `TRUE` but uses better name
- `TRUE`(deprecated): same behavior as current to keep compatibility, but DON'T suggest to use anymore 
- `zip`: forcely use `zip`           
- `OFF` or `FALSE`: disable the packaging     
- others: DON'T accept

How's your opinion?

Reply:
I have most of what you suggested implemented. I'm still working out some bugs with the logic. I don't do much shell scripting.

Reply:
I believe I have worked out the logic. I have tested `compress_assets` with the values `zip`, `auto`, `off` and it appears to be working.

Reply:
It looks great! Thanks!